ReadMe.md

```diff
@@ -0,0 +1,77 @@
+# **STRONGR** (**Workout Tracker**)
+
+**STRONGR** is a workout journal for the connected age. Schedule future workouts, review past workouts, and even bring STRONGR into the gym with you to track your performance on every exercise in your routine.
+
+# DISCLAIMER - STRONGR IS UNDER ACTIVE DEVELOPMENT AT LAMBDA LABS. THIS REPO IS NOT FEATURE COMPLETE.
+
+## Building Locally
+  The repo contains two main directories: `server` and `client`. You will need to build each to run the app for local development. Commands for each directory assume you start at the route of that directory.
+  
+  Server:
+```npm install && nodemon server.js``` 
+
+  Client:
+```npm install && npm start```
+
+## Deploying
+  The standard method for deploying the app is Zeit's [Now](https://zeit.co/now) service, and the `package.json`s in each main directory provide `now-start` scripts for automatic deploy configuration. Note that the `dotenv` code in the repo requires that for the backend/server, either the config vars be set through the Now app or CLI, or that a `.env` be included with the deploy through the `--dotenv` flag.
+
+  A daily deploy is currently being maintained at `strongr.tech` (and `strongr-server.tech` for the back-end).
+
+  ### Config Variables:
+  > USERNAME should be a username for an mLab database.
+
+  > PASSWORD should correspond to USERNAME
+
+  > SECRET is an arbitrary string, used to sign JWTs.
+
+  > SENDGRID_API_KEY is used to send password recovery emails through Sendgrid
+
+  > MAILER_EMAIL_ID is the SendGrid mailer ID, which will determine the email address those emails are sent from.
+
+## API
+
+  * `/register`
+    * { username, password, email }
+    * Creates a new User document and responds with its token.
+
+  * `/login`
+    * { username, password }
+    * Searches for a User document and responds with its token if one is found.
+
+  * `/auto-login`
+    * No request body, but route requires a valid JWT in the "x-access-token" header.
+    * Checks a provided token for validity, responding with the token and an OK status if the token is good. For an example of usage, see the `AccessControl` component.
+
+  * `/forgot_password`
+    * { username, email }
```

Comment:
This is more of a reminder for me to go fix this, you shouldn't need the username, so that means I need to go update some code.